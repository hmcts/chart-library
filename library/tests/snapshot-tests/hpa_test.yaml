suite: Snapshot tests
templates:
  - templates/v2/hpa.yaml
tests:
  - it: manifest should match snapshot
    asserts:
      - matchSnapshot: {}