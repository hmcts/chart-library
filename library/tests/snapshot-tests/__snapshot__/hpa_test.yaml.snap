manifest should match snapshot:
  1: |
    apiVersion: autoscaling/v2
    kind: HorizontalPodAutoscaler
    metadata:
      annotations: null
      labels:
        app.kubernetes.io/instance: RELEASE-NAME
        app.kubernetes.io/managed-by: Helm
        app.kubernetes.io/name: release-name-library
        helm.sh/chart: library-0.1.0
      name: release-name-library
    spec:
      maxReplicas: 5
      metrics:
        - resource:
            name: cpu
            target:
              averageUtilization: 80
              type: Utilization
          type: Resource
        - resource:
            name: memory
            target:
              averageUtilization: 80
              type: Utilization
          type: Resource
      minReplicas: 2
      scaleTargetRef:
        apiVersion: apps/v1
        kind: Deployment
        name: release-name-library
